Snippets/ASB/ASB_7/Performance.cs

```diff
@@ -23,7 +23,7 @@ await endpointInstance.Send(new SomeMessage())
         #endregion
 
         #region asb-fast-send
-        var tasks = new List<Task>();
+        var tasks = new List<Task>(10000);
```

Comment:
Would it make sense to extract the value from here and loop and call it with something to disclose the intention (number-of-messages-to-send)?